.travis.yml

```diff
@@ -24,14 +23,11 @@ script:
 
 before-deploy:
 - echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
-- docker login --username=joleesoftwire --password=$HEROKU_API_KEY registry.heroku.com
 - docker push joleesoftwire/todoapp:latest
-- docker tag joleesoftwire/todoapp registry.heroku.com/jol-todoapp/web
-- docker push registry.heroku.com/jol-todoapp/web
 
 deploy:
   skip_cleanup: true
   provider: script
-  script: heroku container:release --app jol-todoapp web
+  script: curl -dH -X POST $AZURE_URL
```

Comment:
Worth noting that curl doesn't fail the pipeline by default just because it sees a non-200 status code. You might want to also pass the `--fail` flag here so that you have some confidence that Travis reporting success does mean that this call has succeeded